src/perf_client/timing.h

```diff
@@ -163,25 +163,26 @@ namespace timing
     vector<PerRound> per_round;
   };
 
-  static CommitIDs parse_commit_ids(const RpcTlsClient::Response& response)
+  static std::optional<CommitIDs> parse_commit_ids(
```

Comment:
This one is subtle. Because we previously always did some kind of auth check, even if we didn't use/require it, every request resulted in KV reads so got a TxID it could report in the headers (including the read-only stuff). But now our un-authed command endpoints (and all the common ones are now un-authed) will legitimately do _no interaction with the the KV at all_, even in the framework code. So they get no version headers, and some client code that expected these were always present needs to be updated.